libkbfs/folder_branch_ops.go

```diff
@@ -1862,8 +1862,8 @@ func (fbo *folderBranchOps) unembedBlockChanges(
 
 	md.AddRefBytes(uint64(info.EncodedSize))
 	md.AddDiskUsage(uint64(info.EncodedSize))
-	changes.Info = info
 	md.data.cachedChanges = *changes
+	changes.Info = info
```

Comment:
Having it above kept the block info in the cached changes, which could end up confusing things quite a bit.